doc/policies_list/3.3.0/policies.json

```diff
@@ -0,0 +1,1276 @@
+              "properties": {
+                "left": {
+                  "type": "string"
+                },
+                "left_type": {
+                  "default": "plain",
+                  "description": "How to evaluate 'left'",
+                  "oneOf": [
+                    {
+                      "enum": [
+                        "plain"
+                      ],
+                      "title": "Evaluate 'left' as plain text."
+                    },
+                    {
+                      "enum": [
+                        "liquid"
+                      ],
+                      "title": "Evaluate 'left' as liquid."
+                    }
+                  ],
+                  "type": "string"
+                },
+                "op": {
+                  "enum": [
+                    "==",
+                    "!="
+                  ],
+                  "type": "string"
+                },
+                "right": {
+                  "type": "string"
+                },
+                "right_type": {
+                  "default": "plain",
+                  "description": "How to evaluate 'right'",
+                  "oneOf": [
+                    {
+                      "enum": [
+                        "plain"
+                      ],
+                      "title": "Evaluate 'right' as plain text."
+                    },
+                    {
+                      "enum": [
+                        "liquid"
+                      ],
+                      "title": "Evaluate 'right' as liquid."
+                    }
+                  ],
+                  "type": "string"
+                }
+              },
+              "required": [
+                "left",
+                "op",
+                "right"
+              ],
+              "type": "object"
+            }
+          },
+          "properties": {
+            "condition": {
+              "$ref": "#/definitions/condition"
+            },
+            "policy_chain": {
+              "description": "the policy chain to execute when the condition is true",
+              "items": {
+                "type": "object"
+              },
+              "type": "array"
+            }
+          },
+          "required": [
+            "condition"
+          ],
+          "type": "object"
+        },
+        "description": [
+          "Evaluates a condition, and when it's true, it calls its policy chain. ",
+          "This policy cannot be configured from the 3scale UI."
+        ],
+        "name": "Conditional policy",
+        "summary": "Executes a policy chain conditionally.",
+        "version": "builtin"
+      }
+    ],
+    "cors": [
+      {
+        "$schema": "http://apicast.io/policy-v1/schema#manifest#",
+        "configuration": {
+          "properties": {
+            "allow_credentials": {
+              "description": "Whether the request can be made using credentials",
```

Comment:
It looks like something is missing here: _This policy checks whether the request can be made using credentials._
I'm assuming this is a policy; if not, replace the term "policy" with a more appropriate one.